JOSESwift.xcodeproj/project.pbxproj

```diff
@@ -616,6 +617,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		4E06574B223CC78D00D7A199 /* SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = SwiftLint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint autocorrect\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
I'm not sure if we should force the autocorrect in every build step. What would you think of a more passive approach:

- Only linting in the build phase.
- Autocorrect as a Fastlane lane. If we enable danger later, we can let danger remind the user to run the autocorrect lane in case of any linting violations.

My fear with having autocorrect in the build step is that it may also touch things outside the current scope which might make modular commits etc. harder. Let me know what you think and of course, if you disagree!

---

For the warning message, maybe we could add "or run `brew bundle`", since we have `swiftlint` in our `Brewfile`. 👌 🤖

Reply:
I just remembered, we already have a `format_code` lane, no need to add one. 😅 👌 

We run it on every release. 👍

Reply:
Hey Daniel, thanks for your feedback 👌

I'll happily address the issues you mentioned.

Regarding the autocorrect on every build: In our company, we do this and basically never run into an issue.
But I can see where you're coming from, and I'll turn it off 👍

Reply:
Thanks Xaver that sounds great! 🎉